src/codelabs/dart-cheatsheet_migrated.md

```diff
@@ -949,7 +949,7 @@ Their default value is null unless you provide another default value:
 <?code-excerpt "../null_safety_examples/misc/bin/optional_positional_args2.dart"?>
 ```dart
 int sumUpToFive(int a, [int b = 2, int c = 3, int d = 4, int e = 5]) {
-  // ···
+// ···
```

Comment:
unfortunately this is generated by the code-excerpt integration so we can't add two spaces here

Reply:
I'm pretty sure there's a way around that... but I don't remember what it is, so lgtm.